SettingsList.py

```diff
@@ -3281,17 +3281,44 @@ def __init__(self, name, gui_text, min, max, default, step=1,
         name           = 'spawn_positions',
         gui_text       = 'Randomize Overworld Spawns',
         gui_tooltip    = '''\
-            Randomize where you start as Child or Adult when loading
-            a save in the Overworld. This means you may not necessarily
-            spawn inside Link's House or Temple of Time.
+            Randomize where you start when loading
+            a save in the Overworld.
 
             This stays consistent after saving and loading the game again.
         ''',
         default        = False,
+        disable        = {
+            False : {'settings' : ['spawn_positions_age']}
+        },
         shared         = True,
         gui_params     = {
-            'randomize_key': 'randomize_settings',
+            'randomize_key': 'randomize_settings'
+        }
+    ),
+    Combobox(
```

Comment:
The `spawn_positions` and `spawn_positions_age` should be combined into one multiselect combobox. The only reason I could see having separate settings is if there were a "Random" option instead of just None, Child, Adult, and Both.

Reply:
Yeah I see what you mean, but do you think that would look weird, having a combobox in the middle of all the one-way entrance randomization settings? The only reason I did it this way was to hide the combobox if random spawns wasn't select for aesthetics but if you think it would be okay, I'll gladly change it back. Just wanted to run that by you first

Reply:
For reference, this is what I mean:

[https://cdn.discordapp.com/attachments/863163347654279198/1027772151513743461/unknown.png](https://cdn.discordapp.com/attachments/863163347654279198/1027772151513743461/unknown.png)

Reply:
I think it's fine. It already hops between control types for the two ways:

![image](https://user-images.githubusercontent.com/19615534/195995028-c993ef05-fa00-487a-a229-f674cc328975.png)

Reply:
okay I'll revert to what it was prior then!